ui.frontend/src/view/index.js

```diff
@@ -18,6 +18,7 @@ import FormField from "./FormField.js";
 import FormFieldBase from "./FormFieldBase.js";
 import FormPanel from "./FormPanel.js";
 import FormTabs from "./FormTabs";
+import FormCheckBox from "./FormCheckBox";
```

Comment:
Query: Why last 2 imports do not have .js - both work? still, can we have a consistency?

Reply:
Including the .js extension when importing JavaScript modules is unnecessary and not recommended in ES6 syntax. We can change it in top four lines.

Reply:
@egmox You mentioned - not recommended in ES6 syntax
Any reference for the same.
Reference I found which suggest otherwise:

- https://nodejs.org/api/esm.html#mandatory-file-extensions
- https://youtrack.jetbrains.com/issue/WEB-52450/Use-file-extension-in-imports-if-type-module-is-used-in-package.json

Reply:
These are the reference I followed.
https://medium.com/swlh/javascript-best-practices-module-imports-and-generators-8f593f681616
https://chat.openai.com/share/52a95f82-01e8-42c0-8ca8-069ca94f66f9

Reply:
since it has conflicting suggestions from sites, lets have consistency in the code, and follow the existing code practice - to include the extension.